tests/unit/EffectiveDateTime.spec.ts

```diff
@@ -31,58 +31,48 @@ function getLastEvent (wrapper: Wrapper<EffectiveDateTime>, name: string): any {
 }
 
 describe('Effective Date Time component', () => {
-  let wrapperFactory: any
+  let wrapper: any
   const now = new Date()
 
   // invalid future effective date is more than 10 days from now
   const futureInvalidDate = new Date(now.getTime() + 11 * 24 * 60 * 60 * 1000)
 
   const dateTimeDefault = {
-    valid: false,
     isFutureEffective: false,
     dateTimeString: null
   }
 
   const dateTimeValid = {
-    valid: false,
     isFutureEffective: true,
     // effective date is 5 days from now
     dateTimeString: new Date(now.getTime() + 5 * 24 * 60 * 60 * 1000).toISOString()
   }
 
   const dateTimeUnder = {
-    valid: false,
     isFutureEffective: true,
     // effective date is less than 3 minutes from now
     dateTimeString: new Date(now.getTime()).toISOString()
   }
 
   const dateTimeOver = {
-    valid: false,
     isFutureEffective: true,
     dateTimeString: futureInvalidDate.toISOString()
   }
 
   beforeAll(() => {
     // init store
     store.stateModel.currentJsDate = now
+
+    wrapper = mount(EffectiveDateTime, { vuetify })
   })
 
-  beforeEach(() => {
-    wrapperFactory = (stateValues) => {
-      // apply state values
-      Object.keys(stateValues).forEach((key) => {
-        store.stateModel[key] = stateValues[key]
-      })
-      return mount(EffectiveDateTime, { vuetify })
-    }
+  afterAll(() => {
+    wrapper.destroy()
   })
 
-  it('confirms no default Date-Time Selection', () => {
-    const wrapper = wrapperFactory({
-      currentJsDate: now,
-      effectiveDateTime: dateTimeDefault
-    })
+  it('confirms no default Date-Time Selection', async () => {
+    store.stateModel.effectiveDateTime = dateTimeDefault
+    await flushPromises() // wait a bit for validation to complete
```

Comment:
Refactored because we only need the wrapper once, and this is cleaner and easier to understand (IMO).